compiler/luci-interpreter/src/kernels/CMakeLists.txt

```diff
@@ -29,6 +29,8 @@ set(SOURCES
     FullyConnected.cpp
     LeakyRelu.h
     LeakyRelu.cpp
+    LocalResponseNormalize.h
```

Comment:
Looks like it should be `LocalResponseNormalization`.

Reply:
Okay, rename to LocalResponseNormalization.